aikau/src/main/resources/alfresco/lists/views/layouts/Carousel.js

```diff
@@ -372,9 +374,9 @@ define(["dojo/_base/declare",
        * @param {object} payload
        */
       selectItem: function alfresco_lists_views_layouts_Carousel__item(payload) {
-         if (payload.index != null && !isNaN(parseInt(payload.index)))
+         if ((payload.index || payload.index === 0) && !isNaN(parseInt(payload.index, 10)))
```

Comment:
Maybe have ...

```
var targetIndex = payload.index && parseInt(payload.index, 10);
if(!isNaN(targetIndex)) {
```

... which means you can also delete the line below as well.